code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -338,6 +338,25 @@
 	force = WEAPON_FORCE_ROBUST
 	w_class = ITEM_SIZE_NORMAL
 
+/obj/item/tool/sword/cleaver
+	name = "Cleaver Sustine"
+	desc = "A weapon designed by the Hunting Lodge, this massive sword is especially effective against Tengolos, berserk or not."
+	icon = 'icons/obj/weapons-blades.dmi'
+	icon_state = "cleaver"
+	item_state = "cleaver"
+	tool_qualities = list(QUALITY_CUTTING = 30)
+	force = WEAPON_FORCE_BRUTAL
+	armor_penetration = ARMOR_PEN_SHALLOW
+	w_class = ITEM_SIZE_BULKY
+	var/list/effective_faction = list("tengo", "tengolo_berserker") // Which faction the cleaver is effective against.
```

Comment:
Build this into base obj please

Reply:
You mean the `effective_faction` ?

Reply:
the damage mulitplyer and the idea of faction effect.
got a few ideas of how to use this on other object/tools

Reply:
Alright.

Reply:
And done~